package.json

```diff
@@ -3,12 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@types/d3": "^5.7.2",
+    "d3": "^5.16.0",
     "react": "16.13.1",
     "react-dom": "16.13.1"
   },
   "devDependencies": {
     "@types/react": "16.9.34",
     "@types/react-dom": "16.9.7",
+    "prettier": "2.0.5",
```

Comment:
I agree that we should use Prettier, but it would be best to add it, configure it, and set up a linting step in a separate PR.